CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -1325,11 +1325,13 @@ CF_SWIFT_EXPORT void _CFThreadSetName(const char *_Nullable name) {
 }
 
 CF_SWIFT_EXPORT int _CFThreadGetName(char *buf, int length) {
+#if !TARGET_OS_ANDROID
```

Comment:
Rather than wrapping this in an `if !` construct, why not add an `#elif TARGET_OS_ANDROID` above the `#elif DEPLOYMENT_TARGET_LINUX` line? Then there's a sensible place to add implementations in the future should it be needed.

Also, do you know why the variable is TARGET_OS_ANDROID rather than DEPLOYMENT_TARGET_ANDROID?